Modules/Shop/Http/Services/ShopService.php

```diff
@@ -510,13 +507,13 @@ public function getProduct(int $productId): array
             }
             $data['images'] = $images;
             $data['videos'] = $productVideos;
-            $productVariations = ProductVariation::where('product_id', $productDetails->id)->where('status', '1')->get();
+            $productVariations = Product::find($productDetails->id)->productVariations()->where('status', '1')->get();
             $productPrepacks = ProductPrepack::where('product_id', $productDetails->id)->where('active', '1')->get();
```

Comment:
Get the relation productVariations and productPrepacks when you get the $products using eager loading and then instead of query to DB, get the required data from that relations.